src/main/de/linearbits/preferences/PreferenceBoolean.java

```diff
@@ -18,15 +18,29 @@
  */
 public abstract class PreferenceBoolean extends Preference<Boolean> {
 
+	// view or hide the label in chkBox
+	private Boolean hideLabel = false;
```

Comment:
Intendation is off here. Please fix (autoformat could help. Hint: apply only to changed lines).